cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -358,4 +362,7 @@ public String createLog4jProperties(OrderedProperties properties) {
         return super.createLog4jProperties(properties);
     }
 
+    public void setKafkaVersion(String kafkaVersion) {
```

Comment:
Do we really need the setter? Can't we just set the variable in `fromCrd`? If you have a setter, should we have a getter?